model/swagger.yml

```diff
@@ -132,17 +88,14 @@ paths:
         '501':
           $ref: "#/components/responses/Error"
     parameters:
-      - name: identifier
-        in: path
-        description: lidvid or lid
-        required: true
-        schema:
-          type: string
+      - $ref: "#/components/parameters/Fields"
+      - $ref: "#/components/parameters/Identified"
```

Comment:
It does not sounds like a typo "Identified" instead of "Indentifier" because it is used in different places, I am just wondering why ?

Reply:
It was a typo that cut-n-paste (evil pattern) replicated everywhere. Fixed.